### ISSUE TITLE
🐞[v2] Strange behavior with breakpoints

### ISSUE THREAD

**Issue by oskar-gmerek** [master-co/css#111]
### What happened?

This works as expected:

```
<div class="hidden display:flex@sm  ">
```

This not work as expected (element is always hidden)
```
<div class="hidden flex@sm  ">
```

Below is not a bug, but I think that `display:hidden` should be recognised as `display:none` to keep the format.

This not work as expected (element is never hidden)
```
<div class="display:hidden p:15 flex@sm  gap:5  ">
```
same with:
```
<div class="display:hidden p:15 display:flex@sm  gap:5  ">
```

### Version

2.0.0-beta.96

### What browsers are you seeing the problem on?

Chrome, Safari

**Comment by 1aron** [comment]
@oskar-gmerek We found `flex@sm` working fine in JIT but missing in AOT, will fix it asap, thanks for the report!

**Comment by ansarizafar** [comment]
I think scale() is also missing in AOT.

**Comment by 1aron** [comment]
The issue has been fixed in [v2.0.0-beta.99](https://github.com/master-co/css/releases/tag/v2.0.0-beta.99)
